src/ucs/arch/aarch64/global_opts.h

```diff
@@ -0,0 +1,23 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_AARCH64_GLOBAL_OPTS_H_
+#define UCS_AARCH64_GLOBAL_OPTS_H_
+
+#include <ucs/sys/compiler_def.h>
+
+BEGIN_C_DECLS
+
+#define ucs_global_opts_arch_initializer {}
+
+/* built-in memcpy config */
+typedef struct ucs_global_opts_arch_config {
```

Comment:
rename to:
ucs_arch_global_opts / ucs_arch_global_opts_t

Reply:
renamed